Arduino/TemperatureFakeData/TemperatureFakeData.ino

```diff
@@ -55,14 +55,45 @@ void  setup()
 {
   Serial.begin(115200);
   inString = "";
-  printSerial();
+  // printSerial();
```

Comment:
We should probably not remove it. But I had some problems with it because the Arduino Nano which I used for testing did not have serial number etc. burned into the EEPROM. Thus it failed.